provider/techstack/index.test.ts

```diff
@@ -0,0 +1,77 @@
+
+import { describe, expect, test } from 'vitest'
+import {type AnnotationsResult, type CapabilitiesResult} from '@opencodegraph/provider'
+
+import techstack, { type Settings } from './index'
+
+describe('techstack', () => {
+    const SETTINGS: Settings = {
+        yaml: './examples/mazure.yml'
+    }
+
+    test('capabilities', () => {
+        const result = techstack.capabilities({}, SETTINGS)
+
+        expect(result).toBeDefined()
+        expect(result).toStrictEqual<CapabilitiesResult>({
```

Comment:
I think we'd want the techstack file to show up not in code files but in config and build files, so maybe something like:

- `.*` (dotfiles)
- `**/package.json`
- `**/*.bazel`
- `**/README.md`
- `techstack.y?(a)ml` of course
- others like that?

Reply:
Oh, or is the intent to show it alongside the `require` or `import` statements in JS/TS files? Can you add a README describing the expected behavior briefly?

Reply:
Yes, that was the intention for the first draft implementation(only imports), it would be too complex and noisy to annotate rest of the code. 

Annotating build and config files also makes sense, but I'd like to introduce that in a future iteration. I've intentionally kept this first draft minimal. Does this approach sound okay?